.gitignore

```diff
@@ -37,6 +37,7 @@ platform/windows/godot_res.res
 
 # Generated by Godot binary
 .import/
+gdnative_interface.h
```

Comment:
Should be an absolute path (start with `/`), otherwise we're now ignoring `core/extension/gdnative_interface.h`

Reply:
Ah nice, I was not 100% sure about that. Thanks!